public/locales/en.json

```diff
@@ -151,8 +151,8 @@
         "accessTokens": {
             "title": "Access tokens",
             "info": "Access tokens can be used to grant the CLI or third-party applications access to your account.",
```

Comment:
I still don't like this sentence. It seems weird..
How about a more descriptive sentence like
`Access tokens are an password-less authentication method for the CLI or third-party applications.`

Reply:
Changed